hubspot3/associations.py

```diff
@@ -135,6 +137,15 @@ def get_company_to_contacts(self, company_id, params=None, **options):
             **options,
         )
 
+    def get_company_to_deals(self, company_id, params=None, **options):
```

Comment:
I feel the name is a bit strange, why not `get_company_deals`? :)

Reply:
I dont know. I understand what you mean but I named it like this to stay close to the other methods declared in this file:
```
get_deal_to_lines_items
get_company_to_contacts
```

As we are in the `associations` client and as the actual call is to retrieve the `company_to_deal` associations I think that it make sens.

It is very close to the wording used in the hubspot documentation:
https://developers.hubspot.com/docs/methods/crm-associations/crm-associations-overview